tap_fixerio.py

```diff
@@ -100,8 +102,8 @@ def main():
 
     start_date = state.get('start_date',
                            config.get('start_date', datetime.utcnow().strftime(DATE_FORMAT)))
-
-    do_sync(config.get('base', 'USD'), start_date)
+    access_key = state.get('access_key', config.get('access_key'))
```

Comment:
Might be worth doing a bit of hygiene checking here since the config used to not require an `access_key` so that a nice error message is emitted.

Reply:
See lines 107-110